cmp/tile/TilingContainerModel.js

```diff
@@ -0,0 +1,219 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2021 Extremely Heavy Industries Inc.
+ */
+import {HoistModel} from '@xh/hoist/core';
+import {observable, computed, action, makeObservable} from '@xh/hoist/mobx';
+import {isEmpty} from 'lodash';
+
+/**
+ * Model for a TilingContainer.
+ *
```

Comment:
Thinking about how this relates to DataView, and whether we would want to bind it to a store.....